packages/web-twig/tests/components-fixtures/textFieldDefault.twig

```diff
@@ -1,3 +1,5 @@
+{% set inputProps = {"data-validate": "true"} %}
```

Comment:
```suggestion
{% set inputProps = { "data-validate": "true" } %}
```
everywhere? This looks nicer and we do it like this everywhere.

Reply:
Ok, fixed.